chrome-launcher/chrome-finder.ts

```diff
@@ -46,25 +52,43 @@ export function darwin() {
     {regex: /^\/Applications\/.*Chrome.app/, weight: 100},
     {regex: /^\/Applications\/.*Chrome Canary.app/, weight: 101},
     {regex: /^\/Volumes\/.*Chrome.app/, weight: -2},
-    {regex: /^\/Volumes\/.*Chrome Canary.app/, weight: -1}
+    {regex: /^\/Volumes\/.*Chrome Canary.app/, weight: -1},
+    {regex: new RegExp(process.env.LIGHTHOUSE_CHROMIUM_PATH), weight: 150},
+    {regex: new RegExp(process.env.CHROME_PATH), weight: 151}
   ];
   // clang-format on
 
   return sort(installations, priorities);
 }
 
+function resolveChromePath() {
+  if (canAccess(process.env.CHROME_PATH)) {
+    return process.env.CHROME_PATH as string;
```

Comment:
gotta run, but ya that looks safe:

![image](https://user-images.githubusercontent.com/39191/28094466-9f358348-6651-11e7-800b-f9b9c89e424c.png)